pom.xml

```diff
@@ -10,7 +10,7 @@
   <artifactId>plugin</artifactId>
   <!-- Baseline Jenkins version you use to build and test the plugin. Users 
    must have this version or newer to run. -->
-  <version>2.8</version>
+  <version>1.609.3</version>
```

Comment:
To specify the Jenkins version to test against, you would update jenkins.version below. This version drive the parent pom now.